os/board/rtl8721csm/src/rtl8721csm_boot.c

```diff
@@ -85,6 +90,47 @@ extern FAR struct gpio_lowerhalf_s *amebad_gpio_lowerhalf(u32 pinname, u32 pinmo
 /************************************************************************************
  * Public Functions
  ************************************************************************************/
+void board_i2s_initialize(void)
+{
+#if defined(CONFIG_AUDIO_I2SCHAR) && defined(CONFIG_AMEBAD_I2S)
+	i2schar_devinit();
+#endif
+}
+
+void board_spi_initialize(void)
+{
+#ifdef CONFIG_SPI
+	struct spi_dev_s *spi;
+	spi = up_spiinitialize(1);
+
+#ifdef CONFIG_SPI_USERIO
+	if (spi_uioregister(1, spi) < 0) {
+		lldbg("Failed to register SPI%d\n", 1);
```

Comment:
If we only use the `SPI1` only, we don't need to use `%d`.

Reply:
Done.